java/pom.xml

```diff
@@ -35,7 +35,7 @@
     <netty.version>4.1.52.Final</netty.version>
     <gson.version>2.2.4</gson.version>
     <jackson.version>2.11.3</jackson.version>
-    <vertx.version>3.9.5</vertx.version>
+    <vertx.version>4.1.2</vertx.version>
```

Comment:
I think it is better to upgrade to the latest version of Vertx and its declared dependencies:
* Vertx: `4.2.6`
* Netty: `4.1.74.Final`
* Jackson: `2.13.1`